chessboard/board.py

```diff
@@ -70,21 +69,13 @@ def __eq__(self, other):
         return all(self.occupied_co[color] == other.occupied_co[color]
                    for color in (BLACK, WHITE))
 
-    def __ne__(self, other):
```

Comment:
I think we need to keep this method. If we don't, `boardA != boardB` will return a different value than `not (boardA == boardB)`, since `__eq__` overrides `==` but not `!=`